Jenkinsfile

```diff
@@ -1145,6 +1149,10 @@ pipeline {
                                                test_tag=pr,hw,small
                                            fi
                                            tnodes=$(echo $NODELIST | cut -d ',' -f 1-3)
+                                           clush -B -S -o '-i ci_key' -l root -w ${tnodes} \
+                                             "if [ -e /sys/class/net/ib0 ]; then ifconfig ib0; ifup ib0; fi"
+                                           clush -B -S -o '-i ci_key' -l root -w ${tnodes} \
+                                             "if [ -e /sys/class/net/ib1 ]; then ifconfig ib1; ifup ib1; fi"
```

Comment:
These can be combined to cut ssh overhead in half:
```suggestion
                                           clush -B -S -o '-i ci_key' -l root -w ${tnodes} \
                                             "for i in 0 1; do
                                                 if [ -e /sys/class/net/ib\\\$i ]; then
                                                     ifconfig ib\\\$i; ifup ib\\\$i
                                                 fi
                                             done"
```

It would be good to gather some diagnostics when this needs doing, in an effort to fix the root issue instead of just starting a pile of band-aids.

```suggestion
                                           clush -B -S -o '-i ci_key' -l root -w ${tnodes} \
                                             "set -x
                                             for i in 0 1; do
                                                 if [ -e /sys/class/net/ib\\\$i ]; then
                                                     {
                                                         echo \"Found interface ib\\\$i down after reboot on \\\$HOSTNAME\"
                                                         ifconfig ib\\\$
                                                         systemctl status
                                                         systemctl --failed
                                                         journalct -n 500
                                                         ifconfig ib\\\$i
                                                         ifup ib\\\$i
                                                     | | mail -s \"Interface found down after reboot\" esad-ci-team@intel.com
                                                 fi
                                             done"
```

where `esad-ci-team@intel.com` is some alias for you and I (and mark).